LoopKitUI/View Controllers/AddEditOverrideTableViewController.swift

```diff
@@ -0,0 +1,643 @@
+        let keyboard = OverrideSymbolInputController()
+        keyboard.delegate = self
+        return keyboard
+    }()
+
+    public override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        switch Section(rawValue: indexPath.section)! {
+        case .properties:
+            switch propertyRow(for: indexPath) {
+            case .symbol:
+                let cell = tableView.dequeueReusableCell(withIdentifier: LabeledTextFieldTableViewCell.className, for: indexPath) as! LabeledTextFieldTableViewCell
+                cell.titleLabel.text = NSLocalizedString("Symbol", comment: "The text for the override preset symbol setting")
+                cell.textField.text = symbol
+                cell.textField.placeholder = SettingsTableViewCell.NoValueString
+                cell.customInput = overrideSymbolKeyboard
+                cell.delegate = self
+                return cell
+            case .name:
+                let cell = tableView.dequeueReusableCell(withIdentifier: LabeledTextFieldTableViewCell.className, for: indexPath) as! LabeledTextFieldTableViewCell
+                cell.titleLabel.text = NSLocalizedString("Name", comment: "The text for the override preset name setting")
+                cell.textField.text = name
+                cell.textField.placeholder = NSLocalizedString("Running", comment: "The text for the override preset name field placeholder")
+                cell.delegate = self
+                return cell
+            case .insulinNeeds:
+                let cell = tableView.dequeueReusableCell(withIdentifier: InsulinSensitivityScalingTableViewCell.className, for: indexPath) as! InsulinSensitivityScalingTableViewCell
+                cell.scaleFactor = insulinNeedsScaleFactor
+                cell.delegate = self
+                return cell
+            case .targetRange:
+                let cell = tableView.dequeueReusableCell(withIdentifier: DoubleRangeTableViewCell.className, for: indexPath) as! DoubleRangeTableViewCell
+                cell.numberFormatter = quantityFormatter.numberFormatter
+                cell.titleLabel.text = NSLocalizedString("Target Range", comment: "The text for the override target range setting")
+                cell.range = targetRange
+                cell.unitLabel.text = quantityFormatter.string(from: glucoseUnit)
+                cell.delegate = self
+                return cell
+            case .startDate:
+                let cell = tableView.dequeueReusableCell(withIdentifier: DateAndDurationTableViewCell.className, for: indexPath) as! DateAndDurationTableViewCell
+                cell.titleLabel.text = NSLocalizedString("Start Time", comment: "The text for the override start time")
+                cell.datePicker.datePickerMode = .dateAndTime
+                cell.datePicker.minimumDate = min(startDate, Date())
+                cell.datePicker.maximumDate = Date() + CarbStore.defaultMaximumAbsorptionTimeInterval
```

Comment:
I believe my original intent was to limit confusion, since any override more than `CarbStore.defaultMaximumAbsorptionTimeInterval` into the future wouldn't have an immediate effect—but I no longer think it makes sense to restrict this. Someone will likely want to e.g. schedule an override for their morning run the night before.